packages/xo-server-test/src/_xoConnection.js

```diff
@@ -278,7 +279,11 @@ afterAll(async () => {
   await xo.close()
   xo = null
 })
-afterEach(() => xo.deleteTempResources())
+afterEach(async () => {
+  jest.setTimeout(parseDuration(config.deleteTempResourcesTimeout))
```

Comment:
Add a comment explaining this line, ie why a timeout is set.

Reply:
The comment [here](https://github.com/vatesfr/xen-orchestra/pull/5117/files#diff-83711d7804153ebdd786b4a6cf673d1cR5) is not enough.?

Reply:
I think it would make more sense in the code itself because it would make easier to understand that the `setTimeout` here is not to limit the duration of the `afterEach` but on the contrary to avoid it from being interrupted by the default timeout.

But it's good enough as it is.